features/google/src/popUps/info/base.jsx

```diff
@@ -1,16 +1,20 @@
 import React from "react";
 import infoPopUps from "./";
 import { SideSheet, SideSwiper } from "@polypoly-eu/poly-look";
+import i18n from "!silly-i18n";
+
+import "./base.css";
 
 const BaseInfoPopUp = ({ onClose, name }) => {
     return (
         <SideSwiper
             onClose={onClose}
             open={true}
             lastChildSelector=".poly-button"
+            leftDistance="25vw"
             Component={(props) => (
                 <SideSheet
-                    title={"How to read this"}
+                    title={i18n.t("commonInfoScreen:baseInfo.title1")}
                     okLabel={"ok"}
```

Comment:
This should be in upper case, "OK", or preferably use the CSS property `text-transform: uppercase`.

Reply:
Fixed by using translations 😅